yoti-sdk-api/src/main/java/com/yoti/api/client/spi/remote/call/ErrorDetails.java

```diff
@@ -0,0 +1,88 @@
+package com.yoti.api.client.spi.remote.call;
+
+import java.util.Objects;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+
+@JsonDeserialize(builder = ErrorDetails.Builder.class)
+public final class ErrorDetails {
+
+    private final String code;
+    private final String description;
+
+    private ErrorDetails(Builder builder) {
+        this.code = builder.code;
+        this.description = builder.description;
+    }
+
+    public static Builder builder() {
+        return new Builder();
+    }
+
+    public String getCode() {
```

Comment:
Will this always be an integer? Was just thinking it would be easier for clients to switch on, if it's an int